ziti/src/main/kotlin/org/openziti/impl/ZitiContextImpl.kt

```diff
@@ -515,11 +520,7 @@ internal class ZitiContextImpl(internal val id: Identity, enabled: Boolean) : Zi
                 withTimeout(timeout) {
                     serviceUpdates().filter {
                         it.type == ZitiContext.ServiceUpdate.Available &&
-                            servicesByName.get(it.service.name)?.let { svc ->
-                                svc.interceptConfig()?.let {
-                                    it.addresses.any { it.matches(host) } && it.portRanges.any { r -> port in r.low..r.high }
-                                } ?: false
-                            } ?: false
+                                (it.service.interceptConfig()?.matches(host, port) ?: false)
```

Comment:
i imagine we don't support wildcards right? not something to worry about?

Reply:
it does support wildcards. the check was moved to https://github.com/openziti/ziti-sdk-jvm/pull/677/files#diff-f08e5ccd169c44e7b709effdec8410cdbb71bfb96b289f8612ef16c73513dc3bR62